vagrant/Vagrantfile

```diff
@@ -0,0 +1,36 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "generic/ubuntu2004"
+  config.vm.hostname = "script-languages"
+  config.vm.network :forwarded_port, guest: 8888, host: 8888
+  config.vm.network :forwarded_port, guest: 6666, host: 6666
+  config.vm.synced_folder ".", "/home/vagrant/vagrant", type: "rsync", rsync__exclude: ".git/"
+  config.vm.synced_folder "..", "/home/vagrant/script-languages-release", type: "rsync", rsync__exclude: [".git/",".build_output"], rsync__args: ["--verbose", "--archive", "-z", "--links", "--delete"], rsync__verbose: true
```

Comment:
Does this mean the user does not need to call `vagrant rsync-auto`? Or does `vagrant rsync-auto` use this config lines to check what needs to be synced?

Reply:
vagrant rsync and rsync-auto use these config lines to check what needs to be synced